components/discord_bot/actions/modify-channel/modify-channel.mjs

```diff
@@ -0,0 +1,223 @@
+import common from "../common.mjs";
+import constants from "../../constants.mjs";
+
+const { discord } = common.props;
+
+const {
+  GUILD_TEXT,
+  GUILD_NEWS,
+  ...OTHER_CHANNELS
+} = constants.CHANNEL_TYPES;
+
+const NOT_ALLOWED_CHANNELS =
+  Object.keys(OTHER_CHANNELS).map((key) => OTHER_CHANNELS[key]);
+
+export default {
+  ...common,
+  key: "discord_bot-modify-channel",
+  name: "Modify Channel",
+  description: "Update a channel's settings. [See the docs here](https://discord.com/developers/docs/resources/channel#modify-channel)",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    ...common.props,
+    channelId: {
+      description: "Please select the channel you'd like to change its name",
+      propDefinition: [
+        discord,
+        "channelId",
+        ({ guildId }) => ({
+          guildId,
+          notAllowedChannels: NOT_ALLOWED_CHANNELS,
+        }),
+      ],
+    },
```

Comment:
You can leave out `channelId` here since it's already included in common.mjs

Reply:
Yeah same here since `notAllowedChannels` and `description` are a bit different so I need to rewrite